master/pkg/actor/api/api.go

```diff
@@ -46,8 +54,16 @@ func handleWSRequest(system *actor.System, ctx echo.Context) error {
 	}
 }
 
-func handleRequest(system *actor.System, ctx echo.Context) error {
-	switch resp := system.AskAt(parseAddr(ctx.Request().URL.Path), ctx); {
+func handleRequest(system *actor.System, recipient *actor.Ref, ctx echo.Context) error {
+	var addr actor.Address
+	switch {
+	case recipient != nil:
```

Comment:
non-blocking: Eh, this seems like a place where just `if` would be nicer (also above).

Reply:
done